### DISCUSSION TITLE
Disable modal popup when disconnect

### DISCUSSION THREAD

**Discussion by callmejm** [rainbow-me/rainbowkit#1112] [General]
Hi, would like to ask how to prevent modal show up when disconnect wallet ?
`"@rainbow-me/rainbowkit": "^0.12.4"`
`"wagmi": "^0.12.6"`

I think this issue need to be concern, it has been exist quite some time https://github.com/rainbow-me/rainbowkit/issues/686, it affect our user experience

https://user-images.githubusercontent.com/6957533/227699330-9bd44b93-8317-406f-a7ec-ee9d77740367.mp4

**Comment by myr45500** [comment]
Did you solve this issue?, i'm also having it.

**Comment by myr45500** [comment]
This solution even if it worked, i had to revert back because i will have to fix a lot of things that was caused by the react upgrade i’ll keep looking for a hack around, could help you if it doesn’t breake too much things for you.

1-Try upgrading react first,

```
 "react": "^18.2.0",
 "react-dom": "^18.2.0”,
 "@types/react-dom": "^18.0.11",  //if Typescript
```

Then change the way you render. Read this on how to do it https://stackoverflow.com/questions/71668256/deprecation-notice-reactdom-render-is-no-longer-supported-in-react-18


2- if above doesn’t work, still keep the change you did above but downgrade also to 
```
   "wagmi": "^0.6.8"
   "@rainbow-me/rainbowkit": "^0.5.0",
```

**Comment by callmejm** [comment]
there is a [dirty hack](https://github.com/rainbow-me/rainbowkit/issues/686#issuecomment-1295798813)
